src/net/sourceforge/kolmafia/request/AdventureRequest.java

```diff
@@ -1010,6 +1010,8 @@ private static String parseNoncombatEncounter(final String urlString, final Stri
     return parseEncounter(responseText);
   }
 
+  private static final Pattern BOLD_ENCOUNTER = Pattern.compile("<b(?:| [^>]*)>(.*?)</b>");
```

Comment:
optional: instead of an empty alternative, you could just use `(?: [^>]*)?`.

Reply:
At the moment I prefer the empty but we can always change it later